docker-compose.yml

```diff
@@ -50,6 +51,7 @@ services:
             RPC_PASSWORD: password
             RPC_ALLOW_IP: 0.0.0.0/0.0.0.0
             MASTER_NODE_HOST: masternode
+            WAIT_FOR_MASTER: 1
```

Comment:
Wil je hier een andere naam voor gebruiken? Ik wil graag wegblijven van de _master/slave_ terminologie. Daarbij omschrijft dit ook niet wat er in de entrypoint gebeurt, er wordt simpelweg een paar seconden geslapen. Iets als `DELAY_INIT` zou beter zijn.

Het mooiste zou zijn als we de nodes een healthcheck kunnen geven, dan kun je zo lang of kort wachten tot de admin node ready is om connecties te ontvangen, maar helaas hebben we dat nog niet. Ik zag dat in multichain er iets is als `gethealthcheck`, maar dat is alleen beschikbaar voor Multichain Enterprise. Weet jij of we daar zelf iets voor kunnen maken, @kad-floriw?

Reply:
Denk dat er wel iets te maken valt (ping totdat de host:poort bereikbaar is o.i.d. ). Lijkt mij mooi voor later :) Heb de namen aangepast, beter zo idd. Het is eigenlijk de entry-node i.p.v. master-node

Reply:
Thanks! 

Masternode zouden we idd ook nog kunnen vervangen, dat kan prima in een andere PR. Weet nog niet precies wat een betere naam zou zijn. Gezien die eigenlijk alleen nodig is voor het aanmaken van de chain wellicht `init-node`, `starting-node`, `genesis-node` of idd `entry-node`? Of een extra variabele die de chain start zodat we ze samen kunnen voegen